### DISCUSSION TITLE
Left slide, right slide左滑，右滑

### DISCUSSION THREAD

**Discussion by li1422665717** [jocmp/capyreader#415] [Feature Requests]
Are there any plans to do left swipe on the previous article and right swipe on the next article in the future
后续有没有计划做左滑上一篇文章，右滑下一篇文章

**Comment by jocmp** [comment]
Hey @li1422665717 - I have no plans to add swipe left/right in the reader. If you have any apps in mind that you think have good left/right navigation I'll take a look, but as of now I'm happy with the top/bottom navigation.

For more context, I tried this out when I was exploring the swipe top/bottom navigation. In the end I decided against it for a few reasons. I tried the Android [paging library](https://developer.android.com/topic/libraries/architecture/paging/v3-overview) and saw some strange behavior when loading multiple web views at once, especially when full content is enabled. I also tried the [swipe-to-dismiss](https://developer.android.com/reference/kotlin/androidx/compose/material3/SwipeToDismissBoxState) approach but I wasn't happy with how the interaction felt.

**Reply by Araly** [reply]
I'm using Nextcloud News and it has some pretty straight forward left right slides. The way it works is if you swipe from the right to the left, the whole page moves (instead of first having a swipe action, then as you release it loads the next page)

no idea how much work it is for that to happen for capyreader though ^^

https://github.com/user-attachments/assets/0d2b7c41-e87c-47e9-9fab-729f47a191f1

**Reply by mtfurlan** [reply]
The ttrss android app also has that sideways sliding behavior, and I really prefer it to vertical gestures

**Reply by jocmp** [reply]
This is on my list to explore again! A lot of users have been requesting it in both reviews and here on GitHub. 

I'm tracking it here

- https://github.com/jocmp/capyreader/issues/670

**Comment by jocmp** [comment]
- https://github.com/jocmp/capyreader/discussions/797
